tests/core/partitioning/test_loop_fallback.cpp

```diff
@@ -53,6 +53,7 @@ TEST(Partitioning, CheckLoopFallbackNoEvalCompilesCorrectly) {
 
   std::vector<torch_tensorrt::core::ir::Input> input_ranges{torch_tensorrt::core::ir::Input({1, 10})};
   torch_tensorrt::core::CompileSpec cfg(input_ranges);
+  cfg.partitioning_info.forced_fallback_operators.push_back("aten::ones_like");
```

Comment:
Why was this added?

Reply:
The test relies on ones_like being unconvertible.